src/gi/overrides/Vast.py

```diff
@@ -0,0 +1,6 @@
+from ..module import get_introspection_module
+
+Vast = get_introspection_module('Vast')
+
+Vast.Array.__getitem__ = lambda self, i: self.get_value(i)
```

Comment:
I mean it is fine as of now because it is the vast.array api that needs to be fixed first.